pkg/syncer/resource_syncer.go

```diff
@@ -290,7 +290,7 @@ func (r *resourceSyncer) GetResource(name, namespace string) (runtime.Object, bo
 }
 
 func (r *resourceSyncer) ListResources() ([]runtime.Object, error) {
-	var retObjects []runtime.Object
+	retObjects := make([]runtime.Object, 0, len(r.store.List()))
```

Comment:
This is inefficient to list it twice just to get the exact capacity.  I think 10 is a sufficient initial capacity.

Reply:
How about this?

`rStoreList := r.store.List()
retObjects := make([]runtime.Object, 0, len(rStoreList))

for _, obj := range rStoreList {`

Reply:
yeah that makes sense although I'd shorten the var name from `rStoreList` -> `list`.